common/workunit/workflow.cpp

```diff
@@ -383,6 +386,7 @@ class CCloneWorkflowItem : public CInterface, implements IRuntimeWorkflowItem
         persistWfid = other->queryPersistWfid();
         scheduledWfid = other->queryScheduledWfid();
         persistCopies = other->queryPersistCopies();
+        persistRefresh = other->queryPersistRefresh();
```

Comment:
Not new, but none of the members of CCloneWorkflowItem are initialized in constructor, which would be cleaner.